src/tools/blast/components/results/BlastResult.tsx

```diff
@@ -151,6 +157,12 @@ const BlastResult = () => {
 
   const [selectedEntries, setSelectedEntries] = useState<string[]>([]);
   const [urlParams, setUrlParams] = useState<URLResultParams>();
+  const [hspDetail, setHspDetail] = useState<BlastHsp>();
```

Comment:
I think `hspDetail`, `hitLength` and `hspDetailPanel` could be consolidated into one hook

Reply:
Done